src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSJobReleaseTaskExecutionInformation.cs

```diff
@@ -0,0 +1,101 @@
+﻿// -----------------------------------------------------------------------------
+﻿//
+﻿// Copyright Microsoft Corporation
+﻿// Licensed under the Apache License, Version 2.0 (the "License");
+﻿// you may not use this file except in compliance with the License.
+﻿// You may obtain a copy of the License at
+﻿// http://www.apache.org/licenses/LICENSE-2.0
+﻿// Unless required by applicable law or agreed to in writing, software
+﻿// distributed under the License is distributed on an "AS IS" BASIS,
+﻿// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+﻿// See the License for the specific language governing permissions and
+﻿// limitations under the License.
+﻿// -----------------------------------------------------------------------------
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.34209
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Batch.Models
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using Microsoft.Azure.Batch;
+    
+    
+    public class PSJobReleaseTaskExecutionInformation
+    {
+        
+        internal Microsoft.Azure.Batch.JobReleaseTaskExecutionInformation omObject;
+        
+        private PSTaskSchedulingError taskSchedulingError;
+        
+        internal PSJobReleaseTaskExecutionInformation(Microsoft.Azure.Batch.JobReleaseTaskExecutionInformation omObject)
+        {
+            if ((omObject == null))
+            {
+                throw new System.ArgumentNullException("omObject");
+            }
+            this.omObject = omObject;
+        }
+        
+        public System.DateTime? EndTime
+        {
+            get
```

Comment:
Sorry, not really seeing any options for output formatting when defining properties: https://msdn.microsoft.com/en-us/library/system.codedom.codememberproperty%28v=vs.110%29.aspx